code/getSampleRate.m

```diff
@@ -0,0 +1,31 @@
+function sratesout = getSampleRate(srates)
+%% input: matrix of sample rates of each packet from TimeDomainData.Json 
```

Comment:
The description of the input, I believe, is incorrect. I searched Box and found no file named TimeDomainData.json. The data that this function needs to run is contained in RawDataTD.json, which then needs vectorize the 'SampleRate' column. I see that this function is called in createTimeDomainTable, but since this is current a standalone file, the inputs and outputs need to be better defined since someone might use it outside the createTimeDomainTable function.

Reply:
Resolved with https://github.com/openmind-consortium/UCSF-rcs-data-analysis/pull/3/commits/bc30ea52008cedb324aa5e3355d4955dde804175